otelcollector/shared/telemetry.go

```diff
@@ -51,31 +51,6 @@ func SetupTelemetry(customEnvironment string) {
 		return
 	}
 
-	// Append export commands to .bashrc file
```

Comment:
Thanks Kaveesh for looking into this. We don't need the below exports in the bashrc file anymore?

Reply:
Yeah, we don't need them anymore. The `SetEnvAndSourceBashrcOrPowershell` function also sets the environment variables in bashrc/powershell based on the os type.